dashboard/client/src/pages/task/TaskPage.tsx

```diff
@@ -255,6 +268,16 @@ const TaskLogs = ({
           },
         ] as const)
       : []),
+    ...(task_log_info === null
+      ? [
+          {
+            title: "Logs",
+            contents:
+              "For async actors, logs of individual tasks are not available. " +
+              'Please use the "other logs" link to access the all the logs of the actor instead.',
```

Comment:
Can we change the copy to something like this?

`Logs of aysnc actor tasks or threaded actor tasks (concurrency>1) are only available as part of the actor logs. Click the "Other logs" link above to access the actor logs.`